wherehows-web/app/templates/components/notifications/partials/_stale-search-alert.hbs

```diff
@@ -1,2 +1,9 @@
-The WhereHows search index isn't up to date. Until that gets fixed, there are two workarounds: <br>
-Search for your dataset on CMON (go/cmon) and click on the browse in WhereHows link OR use the browse view in Wherehows to navigate the dataset tree to your dataset.
+The WhereHows search index is currently out of date.<br>
+Please see
+<a
+ href="https://iwww.corp.linkedin.com/wiki/cf/display/ENGS/Metadata+Annotation+FAQs#MetadataAnnotationFAQs-stale-index"
```

Comment:
Can we create go links instead? This is opensource. Ideally, this should all come from an api, I'll create a task for myself for the next sprint. In the interim, go links are preferred

Reply:
go/metadata/faq#MetadataAnnotationFAQs-stale-index does the trick